daemon/licensing.go

```diff
@@ -0,0 +1,9 @@
+package daemon // import "github.com/docker/docker/daemon"
+
+import (
+	"github.com/docker/docker/api/types"
+)
+
+func (daemon *Daemon) fillLicense(v *types.Info) {
+	v.ProductLicense = "Community Engine"
```

Comment:
Question; will we actually use this string for the Community Engine, or is this just to "not return an empty string"? If we don't need this string, we could consider leaving this empty

Reply:
Hiding the field on community engines could lead to ambiguity/confusion - I think it's better to be explicit.